### ISSUE TITLE
Upsample causes NNAPI model to fail

### ISSUE THREAD

**Issue by rubengrab** [pytorch/pytorch#55903]
## ❓ Converting a model containing the Upsample opperation from Pytorch -> TorchScript -> NNAPI is successful, but fails to run on Android Device.

Can we get a working example of a model containing the Upsample opperation,  succesfully being converted to NNAPI?

#### Environment: pytorch 1.9.0.dev20210413 py3.8_cuda10.2_cudnn7.6.5_0

**Comment by rubengrab** [comment]
I also tried different approaches. When using _torch.jit.script()_ instead of _torch.jit.trace()_, and then convert to NNAPI, I get:
_Exception: Unsupported node kind ('prim::Uninitialized') in node %1413 : Tensor = prim::Uninitialized()_
This node is created for some opperation inside F.interpolate() function.

**Comment by rubengrab** [comment]
Problem was from a concatenation I was doing right after the Upsample operation, in which I was specifying the dim=-1. Turns out that when running a NNAPI model contianint torch.cat([], dim=-1) on and Android device, it fails since axis is not >=0.
